src/components/events/commonEventFields/weaponsTable.tsx

```diff
@@ -24,8 +23,8 @@ const WeaponsTable = ({
   stations,
   event,
   setAttr,
-}: WeaponsProps) => {
-  const handleWeaponSelect = (sta: string, wpn: WEAPONS) => {
+}: Props) => {
+  const handleWeaponSelect = (sta: string, wpn: WeaponType) => {
```

Comment:
handleWeaponSelect  is awfully similar to `handleVersionInput` in versionsTable.tsx 

Can you make it the same func in a global place?

Reply:
Done